pages/dkp/kommander/2.3/release-notes/2.2.0/index.md

```diff
@@ -21,15 +21,15 @@ beta: false
 
 Welcome to D2iQ Kubernetes Platform (DKP) 2.3! This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
 
-DKP 2.3 supports Kubernetes versions between xxxx and xxxx. Any cluster you want to attach using DKP 2.3 must be running a Kubernetes version in this range.
+DKP 2.3 supports Kubernetes versions between 1.21.0 and 1.23.x. Any cluster you want to attach using DKP 2.3 must be running a Kubernetes version in this range.
 
 ### Supported versions
 
 | Kubernetes Support | Version |
 | ------------------ | ------- |
 |**Minimum** | 1.21.0 |
-|**Maximum** | 1.22.x |
-|**Default** | 1.22.0|
+|**Maximum** | 1.23.x |
+|**Default** | 1.23.0|
```

Comment:
What does the default version means in context of the Kommander? I would probably remove this row from Kommander docs. Maybe the Konvoy installs a default Kubernetes version.

Reply:
Since we have the exact same Release notes for Kommander and Konvoy (in light of us switching to DKP-level docs), I think we should keep this here, but I will ask the Konvoy team to upgrade with the correct default version once we know which one it is.